doc/classes/Object.xml

```diff
@@ -334,146 +366,204 @@
 			</description>
 		</method>
 		<method name="get_indexed" qualifiers="const">
 			<return type="Variant" />
 			<param index="0" name="property_path" type="NodePath" />
 			<description>
 				Gets the object's property indexed by the given [param property_path]. The path should be a [NodePath] relative to the current object and can use the colon character ([code]:[/code]) to access nested properties. Examples: [code]"position:x"[/code] or [code]"material:next_pass:blend_mode"[/code].
-				[b]Note:[/b] Even though the method takes [NodePath] argument, it doesn't support actual paths to [Node]s in the scene tree, only colon-separated sub-property paths. For the purpose of nodes, use [method Node.get_node_and_resource] instead.
+				[codeblocks]
+				[gdscript]
+				var node = Node2D.new()
+				node.position = Vector2(5, -10)
+				var a = node.get_indexed("position")   # a is Vector2(5, -10)
+				var b = node.get_indexed("position:y") # b is -10
+				[/gdscript]
+				[csharp]
+				var node = new Node2D();
+				node.Position = new Vector2(5, -10);
+				var a = node.GetIndexed("position");   # a is Vector2(5, -10)
+				var b = node.GetIndexed("position:y"); # b is -10
+				[/csharp]
+				[/codeblocks]
+				[b]Note:[/b] In C#, [param property] must be in snake_case when referring to built-in Godot properties.
+				[b]Note:[/b] This method does not support actual paths to nodes in the [SceneTree], only sub-property paths. In the context of nodes, use [method Node.get_node_and_resource] instead.
 			</description>
 		</method>
 		<method name="get_instance_id" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the object's unique instance ID.
-				This ID can be saved in [EncodedObjectAsID], and can be used to retrieve the object instance with [method @GlobalScope.instance_from_id].
+				Returns the object's unique instance ID. This ID can be saved in [EncodedObjectAsID], and can be used to retrieve this object instance with [method instance_from_id].
 			</description>
 		</method>
 		<method name="get_meta" qualifiers="const">
 			<return type="Variant" />
 			<param index="0" name="name" type="StringName" />
 			<param index="1" name="default" type="Variant" default="null" />
 			<description>
-				Returns the object's metadata entry for the given [param name].
-				Throws error if the entry does not exist, unless [param default] is not [code]null[/code] (in which case the default value will be returned). See also [method has_meta], [method set_meta] and [method remove_meta].
-				[b]Note:[/b] Metadata that has a [param name] starting with an underscore ([code]_[/code]) is considered editor-only. Editor-only metadata is not displayed in the inspector and should not be edited.
+				Returns the object's metadata value for the given entry [param name]. If the entry does not exist and [param default] is omitted, an error is generated. Otherwise, returns [param default].
```

Comment:
It is not technically accurate because it returns whatever **default** is on error, but... `null == null`. It's probably worth mentioning the null regardless...